codex-rs/execpolicy/tests/cp.rs

```diff
@@ -1,3 +1,4 @@
+#![expect(clippy::expect_used)]
```

Comment:
For everything in `execpolicy/tests/`, can we do this consistently (especially since copy/paste is inevitable)? Top of the file seems simplest?